src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -234,4 +234,16 @@
   <data name="PX1026Fix" xml:space="preserve">
     <value>Remove underscores from the DAC declaration</value>
   </data>
+  <data name="PX1027Fix" xml:space="preserve">
```

Comment:
@k-s-popova, please verify this diagnostic and code fix messages

Reply:
@SENya1990, consider to use the following messages:
Remove the forbidden field from the DAC declaration
The {0} field cannot be declared within a DAC declaration
Remove the constructor
Constructors in DACs are prohibited

Reply:
@k-s-popova, thanks